src/mame/snk/hng64_3d.ipp

```diff
@@ -325,6 +325,23 @@ void hng64_state::setCameraProjectionMatrix(const uint16_t* packet)
 	m_projectionMatrix[15] = 0.0f;
 }
 
+void hng64_state::recoverStandardVerts(polygon& currentPoly, int m, uint16_t* chunkOffset_verts, int& counter)
+{
+	currentPoly.vert[m].worldCoords[0] = uToF(chunkOffset_verts[counter++]);
+	currentPoly.vert[m].worldCoords[1] = uToF(chunkOffset_verts[counter++]);
+	currentPoly.vert[m].worldCoords[2] = uToF(chunkOffset_verts[counter++]);
+	currentPoly.vert[m].worldCoords[3] = 1.0f;
+	currentPoly.n = 3;
+
+	uint16_t unused = chunkOffset_verts[counter++]; (void)unused; // chunkOffset_verts[ xxxx+3 ] is set to 0x70 on some 'blended' objects (fatfurwa translucent globe, buriki shadows, but not on fatfurwa shadows)
```

Comment:
Please use the `[[maybe_unused]]` attribute to mark intentionally unused variables.